<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kayit_ol</class>
 <widget class="QWidget" name="kayit_ol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kayıt Ol Ekranı</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 170, 165);</string>
  </property>
  <widget class="QLabel" name="kayit_kullanici_adi_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kullanıcı Adı :</string>
   </property>
  </widget>
  <widget class="QLabel" name="kayt_sifre_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Şifre :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kayit_kullanici_adi_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 85, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kayit_sifre_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 85, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kayit_ol_buton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>Kayıt Ol</string>
   </property>
  </widget>
  <widget class="QLabel" name="durum_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="kayit_ol_durum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
